style.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: aqua;
+}
```

Comment:
- [ ] As requested by the previous reviewer please use the class on the `h1` tag and give style. ie:
`<h1 class="header"> Hello Microverse </h1>`
`header { color: aqua; }`